person_capture/gui_app.py

```
@@ -14112,6 +14139,55 @@ def _write_sdr_crop_from_image_file(
+                return False, "decode_failed"
+            src = np.ascontiguousarray(img[:, :, :3], dtype=np.uint8)
+            smoothed, changed = self._smooth_wic_shadow_noise_bgr(src)
+            if changed <= 0:
+                return True, "", 0
+            ext = ".jpg" if str(path).lower().endswith((".jpg", ".jpeg")) else ".png"
+            tmp = path + f".tmp_wic_shadow_smooth{ext}"
+            try:
+                if os.path.exists(tmp):
+                    os.remove(tmp)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)